### PR TITLE
Multi-driver configs and the other useful fixes

### PR DESCRIPTION
**Pull request by sparshev**
Separated those changes from #66 because the PR become too big.

* Allow for drivers instantiating by using <driver>/<suffix> - really useful when you have a number of AWS projects to combine them on the same node or restrict utilization of machine resources for specific workloads while leaving the rest of the workloads to utilize moar. Docs: https://github.com/adobe/aquarium-fish/wiki/Drivers
* Removed forgotten old integration tests helper

## Related Issue

fixes: #69 

## Motivation and Context

No way we can live without that!

## How Has This Been Tested?

Manually, automatically

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,11 @@ installed on my machine - I can run Fish and it's VMX driver will automatically
 run VMX images. In case I have docker installed too - I can use both for different workloads or
 select the ones I actually want to use by `--drivers` option or via the API.
 
+In case you need to utilize multiple different configuration for the driver - you can use suffix
+for it's name: like `aws` and `aws/dev` - will effectively serve AWS workloads, but both will have
+different keys, so labels will be able to specify `driver: aws/dev` in order to use `aws/dev` config
+or `driver: aws` in order to use `aws` one - so will run in different AWS projects.
```
Comment (svenevs) [inline README.md:147]: The way this reads is a little confusing to me.  Suggestion:

```
In the event you need to use more than one configuration for a given driver, you can add
a suffix `/<name>`.  For example, `aws` and `aws/dev` will both use the AWS driver, but
use a different configuration.  In this example, labels created will need to specify either
`driver: aws` or `driver: aws/dev` to select which configuration to run.
```

**Review by svenevs — CHANGES_REQUESTED** [review]
Refactor looks great to me!  One small suggestion on improving the README verbiage describing the `aws/test` vs `aws/prod` `/<name>` option being introduced here.

File: README.md (hunk 1 above)
Comment (sparshev) [reply README.md]: Yep, just changed and a bit updated README.md to reflect the current state.

**Review by svenevs — APPROVED** [review]
Looks good to me!

**Pull request merged**